File: fapi/src/main/java/com/gym/fapi/models/Faq.java
package com.gym.fapi.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;


@JsonIgnoreProperties(ignoreUnknown = true)
public class Faq {

    private int idFaq;
    private String question;
    private String answer;

    public Faq() {
    }

    public Faq(int idFaq, String question, String answer) {
        this.idFaq = idFaq;
        this.question = question;
        this.answer = answer;
    }

    public int getIdFaq() {
        return idFaq;
    }

    public void setIdFaq(int idFaq) {
        this.idFaq = idFaq;
    }

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }

    public String getAnswer() {
        return answer;
    }

    public void setAnswer(String answer) {
        this.answer = answer;
    }
}

File: frontend/src/app/modules/menu/components/faq/faq.ts
export class Faq{
  idFaq: string;
  question: string;
  answer: string;



  static cloneBase(faq: Faq): Faq {
    const clonedNews: Faq = new Faq();
    clonedNews.idFaq = faq.idFaq;
    clonedNews.question = faq.question;
    clonedNews.answer = faq.answer;
    return clonedNews;
  }
}


